components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -28,6 +30,28 @@ export default function TeamSettings() {
     const [slug, setSlug] = useState(org?.slug || "");
     const [updated, setUpdated] = useState(false);
     const updateOrg = useUpdateOrgMutation();
+    const [teamSetting, setTeamSetting] = useState<TeamSettings>({ teamId: org?.id ?? "", workspaceSharingDisabled: false });
+
+    const loadSetting = () => {
+        if (!org) {
+            return;
+        }
+        getGitpodService().server.getTeamSettings(org.id).then((r) => setTeamSetting(r))
```

Comment:
This should be a query. Maybe @selfcontained can help with feedback here. :pray:

Reply:
Updated, correct me if I used it wrong 🙏  cc @selfcontained